docker/grafana/provisioning/datasources/iml-postgres-datasource.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: 1
```

Comment:
Why do we need this file if we are also copying over the grafana-iml.ini?

Reply:
I'll try removing this one and verify that it is still added to the datasources section. I added this here because it was also manually added in the service config. With that said, if we don't need this, we can probably remove it from the service config as well.

Reply:
After removing this I don't see any errors so it should be ok. The data source shows only the influx entries but for now I can leave it like this and rebase the other patches on top to make sure it still works properly:
![image](https://user-images.githubusercontent.com/1100657/72556746-4517cd80-386d-11ea-88e5-c070523de767.png)